File: 350GameJavaClient/src/cs350project/Command.java
package cs350project;

public enum Command {
    MOVING_LEFT(0x40),
    MOVING_RIGHT(0x20),
    CROUCHING(0x10),
    JUMPING(0x8),
    BLOCKING(0x4),
    PUNCH(0x2),
    HIGH_KICK(0x1),
    LOW_KICK(0x5);
        
    private final int code;
        
    private Command(int code) {
        this.code = code;
    }
    
    public int getCode() {
        return code;
    }
}

File: 350ServerApp/ConsoleApp1/Enumerations/Command.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GameServer.Enumerations
{
    public enum Command
    {
        IDLE = 0,
        MOVING_LEFT = 1,
        LOW_KICK = 2,
        HIGH_KICK = 4,
        PUNCH = 8,
        BLOCKING = 16,
        JUMPING = 32,
        CROUCHING = 64,
        MOVING_RIGHT = 128
    }

    public static class CommandUtilities
    {
        public static Command ParseCommand(byte commadByte)
        {
            return (Command)Enum.Parse(typeof(Command), Enum.GetName(typeof(Command), commadByte));
        }
    }
}
